inc/app_replacer.php

```diff
@@ -31,25 +31,25 @@ abstract class Optml_App_Replacer {
 	 *
 	 * @var array
 	 */
-	protected static $possible_src_attributes = null;
+	protected static $possible_src_attributes = [];
```

Comment:
For this file is the same as for the other variables initialized with `null` I mentioned before.